core/capabilities/transmission/local_target_capability.go

```diff
@@ -15,13 +15,15 @@ import (
 type LocalTargetCapability struct {
 	lggr logger.Logger
 	capabilities.TargetCapability
-	peerID p2ptypes.PeerID
-	don    capabilities.DON
+	capabilityID string
```

Comment:
Added this so we can log the capability ID that is being wrapped. I didn't want to waste a grpc call on `Info` when we already have this information